app/views/spina/admin/articles/_form.html.haml

```diff
@@ -22,6 +22,11 @@
             = render 'spina/admin/shared/rich_text_field', form: f, field: :content
         %tr
           %td
+            Excerpt
+          %td
+            = f.text_area :excerpt
```

Comment:
reason why I am not using the rich_text_field here... there's a bug in Spina, it's expecting the that the form will not be for the same object. Otherwise all the rich_text_field gets field with the same content.
So text_field for now.
But I do wonder if instead of adding fields to the articles table, we should be using "templates" for articles. And have an interface to select different template type for a specific article. Similar to how it's done with the pages